### ISSUE TITLE
Security is bad

### ISSUE THREAD

**Issue by CodeitToGit** [cornflourblue/angular-9-role-based-authorization-example#5]
Because you are saving whole user information (except password but it doesn't matter) in localStorage you can easily change the role inside the token to Admin and then use the app with admin privileges, and localStorage has to contain Role and Token from user object at all times if you want to keep the user logged in and able to refresh the page.

**Comment by TGihan** [comment]
Yes, this is true. after page refresh how we keep logged user. because in this example after page refresh user details getting reset and again need to login to the application.
